sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3853,11 +3906,14 @@ SUBROUTINE MISCLN
                 ENDIF
                 STP=(MLCAPE(I,J)/D1500)*MLLCLtmp*(ESRH(I,J)/150.)*&
                         ESHRtmp*MLCINtmp
+                GRID1(I,J) = SPVAL
+                IF(LLOW(I,J)<spval.and.LUPP(I,J)<spval) THEN
```

Comment:
remove this if block and add to the next if block
IF (STP>0 .and. STP<spval) THEN

Reply:
STP is computed from ESRH and that is computed from LLOW and LUPP, but STP will never be spval.